mlflow/__init__.py

```diff
@@ -40,6 +40,9 @@
 import mlflow.projects as projects  # noqa
 import mlflow.tracking as tracking  # noqa
 import mlflow.tracking.fluent
+from mlflow.utils.logging_utils import _configure_default_logger
+
+_configure_default_logger()
```

Comment:
As mentioned offline, we should figure out how other OSS projects do this to avoid overriding our users' log settings, and to ensure we always configure out logger.

Reply:
Yep! It looks like other projects configure a top-level logger for their project. All submodules of the project then inherit the top-level logging configuration. For example, if we configure a logger for the `mlflow` top-level module name, submodules like `mlflow.sagemaker` will pick up the configuration.

This approach also ensures that the logging configurations of external modules are not affected.